htdocs/bookmarks/bookmarks.lib.php

```diff
@@ -63,7 +63,7 @@ function printDropdownBookmarksList()
 	}
 	if (!empty($_POST) && is_array($_POST)) {
 		foreach ($_POST as $key => $val) {
-			if ((preg_match('/^search_/', $key) || in_array($key, $authorized_var))
+			if ((preg_match('/^search_|^limit$/', $key) || in_array($key, $authorized_var))
```

Comment:
The string 'limit' is already inside the array 
 $authorized_var
so the test should already be true, with no need of this addition. I miss something....